doc/articles/uno-development/debugging-uno-ui.md

```diff
@@ -67,19 +67,6 @@ You may improve your built time by selecting an active target framework, see the
 The Uno solution provides a set of sample applications that provide a way to test features, as
 well as provide a way to write UI Tests. See [this document](working-with-the-samples-apps.md) for more information.
 
-## Using the Package Diff tool
```

Comment:
I think a good rule of thumb here is to remove the body of this section but leave the header ('Using the Package Diff tool') and a redirect like 'See [link] for information on blah blah.' Just to avoid breaking peoples' existing bookmarks. 

(Probably not a big deal in this specific case, but I thought about it when moving the platform-specific XAML info to a standalone file.)

Reply:
Yeah I agree. Eventually this will be retired as the content will be expanded into their own documents. Thus, as each section gets expanded hyperlink and then eventually delete the file.